acrobat/scripts/scripts.js

```diff
@@ -417,13 +417,8 @@ const { ietf } = getLocale(locales);
   loadIms().then(() => {
     const imsIsReady = new CustomEvent('IMS:Ready');
     window.dispatchEvent(imsIsReady);
-  }).catch((e) => {
-    console.error(e);
-  });
-
-  window.addEventListener('IMS:Ready', async () => {
     if (!window.adobeIMS.isSignedInUser()) {
-      document.querySelector('.dropZoneContent').style.pointerEvents = 'auto';
+      widgetBlock.querySelector('.dropZoneContent').style.pointerEvents = 'auto';
```

Comment:
It failed on non-frictionless pages.

https://mwpw-147913-loadims--dc--adobecom.hlx.page/acrobat/business/resources